x-pack/legacy/plugins/apm/cypress/snapshots.js

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
what's this file about?

Reply:
Auto generated by the snapshot module on `yarn install`. And no, we don't need it.

https://github.com/cypress-io/snapshot/blob/master/src/add-initial-snapshot-file.js#L9